test/acceptance/objects/crefs_test.go

```diff
@@ -452,6 +540,21 @@ func TestObjectBatchToClassDetection(t *testing.T) {
 func TestObjectCrefWithoutToClass(t *testing.T) {
```

Comment:
can we also here preserve this test without toClass and a new one that performs this test but with otherClass and tenant as you did here?

Reply:
Not sure if I understood you right. Should I have the old test from before this PR and then in addition the new test?

Reply:
yes, bc `TestBatchRefsWithoutToClass` and `TestObjectCrefWithoutToClass` should pass in it's original state after refactor right? now those 2 tests introduce `otherClassMT` and `tenants`

Reply:
Yes, but I don't change anything about the original classes.

Before this PR the .exist() call was done without the auto-detected toClass. So if there are two objects with the same ID it could grab the wrong one (eg the one from otherClassMT). The tenancy there was only added because in case an object from the wrong class was returned it would error immediately and clearly show the bug.

Everything else about the autodetection is not affected by the addition, eg it works exactly like before